### PR TITLE
Include prebuilt Hot Forge binary

### PR DESCRIPTION
**Pull request by feikesteenbergen**
This significantly reduces the time to build a Docker Image

### CODE REVIEW

**Review by inchuev — APPROVED** [review]

**Comment by inchuev** [comment]
I'm just wondering whether we still need Rust compiler in the image, though.

**Comment by feikesteenbergen** [comment]
@inchuev

> we still need Rust compiler in the image

We do, there's multiple rust components that require it, for example https://github.com/timescale/timescaledb-toolkit

File: Dockerfile [hunk 1]
```diff
@@ -126,6 +126,19 @@ RUN chmod +x /usr/local/bin/docker-entrypoint.sh
...
+# hot-forge is a project that allows hot-patching of postgres containers
+# It is currently a private timescale project and is therefore not included/built by default,
+# and never included in the OSS image.
+ARG TIMESCALE_HOT_FORGE=
+RUN if [ ! -z "${PRIVATE_REPO_TOKEN}" -a -z "${OSS_ONLY}" -a ! -z "${TIMESCALE_HOT_FORGE}" ]; then \
+        GH_REPO="https://api.github.com/repos/timescale/hot-forge/"; \
+        ASSET_ID="$(curl -sL --header "Authorization: token ${PRIVATE_REPO_TOKEN}" "${GH_REPO}/releases/tags/${TIMESCALE_HOT_FORGE}" | jq '.assets[0].id')"; \
+        curl -sL --header "Authorization: token ${PRIVATE_REPO_TOKEN}" \
+                 --header 'Accept: application/octet-stream' \
+                 "${GH_REPO}/releases/assets/40428091" > /usr/local/bin/hot-forge || exit 1; \
```
Comment (alexeyklyukin) [inline Dockerfile:138]: We seem to forget a hard-coded asset ID here @feikesteenbergen ?

Comment (feikesteenbergen) [reply Dockerfile:138]: Ah, sorry about that. Will fix

**Pull request merged**